ta/os_test/pauth_a64.S

```diff
@@ -0,0 +1,16 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2022, Linaro Limited
+ */
+
+#include <asm.S>
+
+FUNC corrupt_pac , :
+	paciasp
+	/* Flip a random bit in PAC */
```

Comment:
I guess this means "Flip a random bit in PAC field of the return address".

Reply:
Yes.